src/tests/VoxelGridTest.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+#include <Corrade/TestSuite/Tester.h>
+#include <Corrade/Utility/Directory.h>
+#include <Magnum/Magnum.h>
+
+#include "esp/geo/VoxelWrapper.h"
+#include "esp/sim/Simulator.h"
+
+#include "configure.h"
+
+namespace Cr = Corrade;
+namespace Mn = Magnum;
+
+// TODO: Add tests for different Sensors
+struct VoxelGrid : Cr::TestSuite::Tester {
+  explicit VoxelGrid();
+
+  void testVoxelGridWithVHACD();
+};
+
+VoxelGrid::VoxelGrid() {
+#ifdef ESP_BUILD_WITH_VHACD
+  addTests({&VoxelGrid::testVoxelGridWithVHACD});
+#endif
+}
+
+#ifdef ESP_BUILD_WITH_VHACD
+void VoxelGrid::testVoxelGridWithVHACD() {
```

Comment:
Some of this PRs functionality (e.g. getters/setters, creating/remove grids, wrapper conversions, global grids, etc...) could be tested without VHACD. Would love to see these added to validate the code as we iterate.